x/cronos/keeper/precompiles/relayer.go

```diff
@@ -63,13 +66,19 @@ func (bc *RelayerContract) Address() common.Address {
 	return relayerContractAddress
 }
 
+func (bc *RelayerContract) SetChainConfig(isHomestead, isIstanbul, isShanghai bool) {
```

Comment:
is it called anywhere?

Reply:
it's called when [runPrecompiledContract](https://github.com/crypto-org-chain/go-ethereum/pull/13/files#diff-8403dffcbec1accad5356c19a66853e9c4cf77a1a11a2d9eeb73ed0c2022257fR278)

Reply:
I see, it seems better to call in ethermint ApplyMessage when init, instead of go-ethereum

Reply:
moved the change to [ethermint](https://github.com/crypto-org-chain/ethermint/pull/380/files#diff-feaf82fa09f2e8d80af866070cc26213ea09ec4c89ae8a122a6bb4153f8450d6R83) instead